_release_common.sh

```diff
@@ -98,9 +98,9 @@ function is_early_product_version_than {
 
 	if [ -n "${ACTUAL_PRODUCT_VERSION}" ]
 	then
-		product_version_1=$(echo "${ACTUAL_PRODUCT_VERSION}" | sed -e "s/-lts//")
+		product_version_1="$(echo "${ACTUAL_PRODUCT_VERSION}" | sed -e "s/-lts//")"
 	else
-		product_version_1=$(_get_product_version | sed -e "s/-lts//")
+		product_version_1="$(_get_product_version | sed -e "s/-lts//")"
```

Comment:
@davisaints, is it really necessary?

When I look in the code base, I see both options, the one without `""` and the one with it. However, I'm seeing much more examples without the quotes. For now, can you please remove the quotes from here?

Thanks